merged_schema/HPXMLMerged.xsd

```diff
@@ -7174,6 +7208,7 @@
 			<xs:enumeration value="garage - unconditioned"/>
 			<xs:enumeration value="ground"/>
 			<xs:enumeration value="living space"/>
+			<xs:enumeration value="manufactured home underbelly"/>
```

Comment:
This enumeration goes on the `<ExteriorAdjacentTo>` element, mostly so floors can specify the underbelly area is below them.